Source/Particles/Collision/BinaryCollision/BinaryCollisionUtils.cpp

```diff
@@ -0,0 +1,64 @@
+/* Copyright 2021 Neil Zaim
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "BinaryCollisionUtils.H"
+
+#include "Particles/MultiParticleContainer.H"
+#include "Particles/WarpXParticleContainer.H"
+
+#include <AMReX_ParmParse.H>
+#include <AMReX_Vector.H>
+
+#include <string>
+
+namespace BinaryCollisionUtils{
+
+    NuclearFusionType get_nuclear_fusion_type (const std::string collision_name,
+                                            MultiParticleContainer* const mypc)
```

Comment:
I think here and in the interface, you can even write
```suggestion
                                               MultiParticleContainer const * const mypc)
```
given that we only call a `const` function on it